contracts/lib/LibPrices.sol

```diff
@@ -68,19 +68,15 @@ library Prices {
         uint256 oldLeverage,
         uint256 newLeverage
     ) public pure returns (uint256) {
-        int256 delta = int256(newLeverage) - int256(oldLeverage);
+        bool leverageHasIncreased = newLeverage > oldLeverage;
```

Comment:
can probably be simplified I think. Should only need one bool it seems, as the only edge case is where `newLeverage = oldLeverage` in which case `newLeveraged - oldLeverage = 0` anyway.

Current way is more explicit, but I think you should just be able to do 

`if (leverageHasIncreased) { add to leverage } else { decrease leverage }`